recipes/log4cxx/all/patches/0002-make-odbc-optional.patch

```diff
@@ -0,0 +1,39 @@
+--- a/src/main/include/CMakeLists.txt
++++ b/src/main/include/CMakeLists.txt
+@@ -85,16 +85,27 @@ include(CheckIncludeFiles)
+ include(CheckIncludeFileCXX)
+ include(CheckLibraryExists)
+ 
+-if(WIN32)
+-	CHECK_INCLUDE_FILES(sqlext.h HAS_ODBC)
+-else()
+-	include(FindPkgConfig)
++option(LOG4CXX_ENABLE_ODBC "Support logging via ODBC" OFF)
```

Comment:
We should not be adding options to build scripts 🤔 this should be removed one the upstream recipe is fixed